dev_guide/expose_service/expose_internal_ip_service.adoc

```diff
@@ -343,16 +343,16 @@ eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
         ...
 ----
 
-. Add a route between the IP address of the host where the master resides and the gateway IP address of the master host:
+. Add a route between the IP address of the host where the master resides and the gateway IP address of the master host. If using a netmask for a networking route, use the `netmask` option, as well as the netmask to use:
 +
 ----
-$ route add -net <host_ip_address> netmask <netmask> gw <gateway_ip_address> dev <device>
+$ route add -host <host_ip_address> netmask <netmask> gw <gateway_ip_address> dev <device>
 ----
 +
 For example:
 +
 ----
-$ route add -host 10.16.41.22 gw 10.16.41.254 dev eth0
+$ route add -host 10.16.41.22 netmask 255.255.255.0 gw 10.16.41.254 dev eth0
```

Comment:
line 342: netmask 255.255.248.0 
So I think here should be '255.255.248.0' instead of 255.255.255.0.